paperqa/docs.py

```diff
@@ -280,6 +280,9 @@ async def aadd(  # noqa: PLR0912
         parse_config = all_settings.parsing
         if dockey is None:
             # md5 sum of file contents (not path!)
+
+            # NOTE: test_get_directory_index_w_manifest fails because now we are getting
```

Comment:
Let's avoid adding mentions of the external test suite into the core package

What makes you add this comment here? Is the test still failing?